src/System.Runtime/tests/System/HashCodeTests.netcoreapp.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+public static class HashCodeTests
+{
+#   if SYSTEM_HASHCODE_TESTVECTORS
```

Comment:
Why are these tests are compiled out?

Reply:
Also - (nit) the normal code formatting is to have `#if FOO` with no spaces between the `#` and `if`.  This will just cause the next person editing this code troubles since VS will attempt to format the code (with the default format settings).

Reply:
@eerhardt they are compiled out because the `GetRandomBytes()` randomization in HashCode removes the determinism. HashCode needs to be adjusted before they can execute correctly.

Reply:
That sounds fine to me.  Any chance you can stick that in a comment in the code?